templates/common/baremetal/files/NetworkManager-resolv-prepender.yaml

```diff
@@ -7,6 +7,10 @@ contents:
     set -eo pipefail
     IFACE=$1
     STATUS=$2
+
+    # Ensure resolv.conf exists before we try to run podman
+    cp /var/run/NetworkManager/resolv.conf /etc/resolv.conf
+
```

Comment:
It's probably safer to do this in the case I think, so it only copies on a valid $STATUS?

We can then also remove the else "Couldn't find a Virtual IP" part

Reply:
Ah yea good call